website/src/controllers/user.js

```diff
@@ -117,12 +117,26 @@ api.score = function(req, res, next) {
     }, saved.toJSON().stats));
 
     // Webhooks
+
+    // Select character data to send
+    var userData = _.pick(user.toJSON(), ['_id', '_tmp', 'stats']); // user.toJSON to copy-by-value
```

Comment:
`_.pick` creates a shallow clone of the object, so on the next line when we go to assign the properties to `userData.stats`, without the `.toJSON()` we would mutate the actual `user.stat` object